lib/v2/cnblogs/pick.js

```diff
@@ -0,0 +1,30 @@
+const got = require('@/utils/got');
```

Comment:
According to
https://github.com/DIYgod/RSSHub/blob/531a756328b2c3c352da235f06d4e1fc3f1fe311/lib/v2/cnblogs/router.js#L6
the file is not used